addon/components/rdf-input-fields/currency-input.hbs

```diff
@@ -0,0 +1,40 @@
+{{#unless @inTable}}
+  <AuLabel
+    @error={{this.hasErrors}}
+    @warning={{this.hasWarnings}}
+    @required={{this.isRequired}}
+    for={{this.inputId}}
+  >
+    {{@field.label}}
+  </AuLabel>
+{{/unless}}
+
+<div class="CurrencyInput">
+  <AuInput
+    @error={{this.hasErrors}}
+    @disabled={{@show}}
+    @width={{if @inTable "block"}}
+    @warning={{this.hasWarnings}}
+    id={{this.inputId}}
+    value={{this.value}}
+    {{on "blur" this.updateValue}}
+    {{au-inputmask
+      options=(hash
+        alias="currency"
+        radixPoint=","
+        groupSeparator="."
+        allowMinus=true
+        inputType="decimal"
+      )
+    }}
+  />
+  <div class="currencyIsoCode">{{this.currencyIsocode}}</div>
```

Comment:
Question: Why was this changed? Is this the better way to display the euro currency or something?

Reply:
The currency symbols could be non-alphanumeric characters. I think we will not be needing other currencies but it keeps us away from any issues with the symbols. The iso code is a standard and can be used for every currency. It has a 3 letter format and is more readable than `€`

`€` or `EUR`
`$` or `USD`

I think if you read this you are also sold on using this :)
`ISO 4217 is a standard published by the International Organization for Standardization that defines alpha codes and numeric codes for the representation of currencies and provides information about the relationships between individual currencies and their minor units.`